docs/data-sources/project.md

```diff
@@ -71,4 +71,30 @@ The following attributes are exported:
 
 * `remove_source_branch_after_merge` - Enable `Delete source branch` option by default for all new merge requests
 
-* `packages_enabled` - Enable packages repository for the project.
\ No newline at end of file
+* `packages_enabled` - Enable packages repository for the project.
+
+* `push_rules` Project push rules (GitLab Starter/Bronze/EE only). Push Rules documented below.
```

Comment:
Can we just say EE maybe? We don't give that level of details elsewhere IIRC.

Also can we link to upstream doc?

Reply:
I searched our docs for "EE" and the only other instance of it I found was in [service_github](https://github.com/gitlabhq/terraform-provider-gitlab/blob/master/docs/resources/service_github.md)
> requires either EE (self-hosted) or Silver and above (GitLab.com).

So in that case it's equally specific. I'm not sure if there are other EE features in the provider that we simply do not mention are EE only? In that case I can remove the mention of EE in the doc.

@nicholasklick You mentioned people were asking for EE features in this provider. Do you think it makes sense for us to document which fields are EE-only, or can we leave that out for simplicity?

Reply:
In my latest push I removed mention of "EE" in the docs change.

Reply:
I also added the doc link to both the data source and the resource. 😉